commons/src/main/java/com/powsybl/commons/config/BaseVoltagesConfig.java

```diff
@@ -66,12 +69,23 @@ public static BaseVoltagesConfig fromInputStream(InputStream configInputStream)
         return yaml.load(configInputStream);
     }
 
-    public static BaseVoltagesConfig fromPath(Path configFile) {
-        Objects.requireNonNull(configFile);
-        try (InputStream configInputStream = Files.newInputStream(configFile)) {
-            return fromInputStream(configInputStream);
-        } catch (IOException e) {
-            throw new UncheckedIOException(e);
+    public static BaseVoltagesConfig fromPath(Path configDir, String configFileName) {
+        Objects.requireNonNull(configDir);
+        Objects.requireNonNull(configFileName);
+        Path configFile = configDir.resolve(configFileName);
+        if (Files.exists(configFile)) {
+            try (InputStream configInputStream = Files.newInputStream(configFile)) {
+                return fromInputStream(configInputStream);
+            } catch (IOException e) {
+                throw new UncheckedIOException(e);
+            }
+        } else {
+            InputStream configInputStream = BaseVoltagesConfig.class.getResourceAsStream("/" + configFileName);
```

Comment:
Could you add a log to tell the user the default config file has been loaded?

Reply:
Done